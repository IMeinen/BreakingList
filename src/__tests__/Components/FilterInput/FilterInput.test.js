import React from 'react';

import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import FilterInput from '../../../components/FilterInput';

Enzyme.configure({ adapter: new Adapter() });
describe('FilterInput component tests', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<FilterInput data={[]} set={console.log} />);
    expect(wrapper).toMatchSnapshot();
  });
});
