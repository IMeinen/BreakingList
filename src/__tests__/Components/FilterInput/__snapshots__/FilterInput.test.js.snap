// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterInput component tests renders correctly 1`] = `
<styled.div>
  <styled.input
    onChange={[Function]}
  />
  <GoSearch
    color="#fff"
    size={24}
  />
</styled.div>
`;
